torch/csrc/jit/passes/remove_mutation.cpp

```diff
@@ -35,6 +35,29 @@ Node* MutationRemover::createSpecialMappedOp(Node* n) {
         graph_->insert(aten::full_like, {inputs.at(0), inputs.at(1)})->node();
   } else if (n->matches("aten::zero_(Tensor(a!) self) -> Tensor(a!)")) {
     new_node = graph_->insert(aten::zeros_like, {n->inputs().at(0)})->node();
+  } else if (
+      n->matches(
+          "aten::normal_(Tensor(a!) self, float mean=0, float std=1, *, Generator? generator=None) -> Tensor(a!)")) {
+    // TODO: we should have normal_like operator
+    // normal(float mean, float std, int[] size, *, Generator? generator=None,
+    // ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool?
+    // pin_memory=None) -> Tensor
+    auto size = graph_->insert(aten::size, {n->inputs().at(0)});
```

Comment:
Is this transformation always on in `RemoveMutation` pass? I feel a little bit uneasy seeing how many extra ops we are adding here?

Reply:
Yes, but `RemoveMutation` isn't always on anywhere

Reply:
I opened issue here: https://github.com/pytorch/pytorch/issues/52334. I dont think it should block landing this bug fix.